packages/login_client/lib/src/login_client.dart

```diff
@@ -107,6 +107,25 @@ class LoginClient extends http.BaseClient {
     }
   }
 
+  /// Restores saved credentials from the credentials storage and set
+  /// `tokenEndpoint`to the one provided in the `oAuthSettings`.
+  Future<Credentials?> get _credentialsToInitialize async {
```

Comment:
We are performing some actions here (call the credentials storage), I'd prefer to make it a method to be more explicit that it hides _some_ complexity